src/Nox.Types/Types/Uri/Uri.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+
+namespace Nox.Types;
+
+public sealed class Uri : ValueObject<System.Uri, Uri> 
+{
+
+    public static Uri From(string value, System.UriKind uriKind = System.UriKind.Absolute)
```

Comment:
I understand that the URI per si already does the validation, so we need to do this in the constructor